src/actions/PaginationAction.js

```diff
@@ -0,0 +1,22 @@
+import actionTypes from "./actionTypes";
+
+export const getMoreArticles = page => dispatch => {
+  return fetch(
+    `https://ah-backend-athena-staging.herokuapp.com/api/articles?limit=5&page=${page}`,
```

Comment:
use an environment variable for the host name.

Reply:
okay, let me work on it right away